packages/gatsby-plugin-canonical-urls/package.json

```diff
@@ -14,11 +14,12 @@
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "license": "MIT",
   "devDependencies": {
-    "babel-cli": "^6.26.0",
+    "@babel/cli": "^7.0.0-beta.38",
+    "@babel/core": "^7.0.0-beta.38",
```

Comment:
the preset is in the root but the build call is local to each package. Its a balance between keeping packages reasonablly independant so different folks can maintain them while also making repo wide stuff a bit easier. so since each package depends on babel/cli core is need b/c peer deps

Reply:
Gotcha, guess that makes sense then.